src/core/push/safari.ts

```diff
@@ -74,6 +110,12 @@ export default class SafariPushManager implements PushOpsManager {
     async requestPermissionAndRegisterForPush(
         ctx: Context
     ): Promise<PushSubscriptionState> {
+        const wasUnregistered = await get<boolean>('wasUnregistered');
```

Comment:
Any value in storing a timestamp of `unregisteredAt` instead of a boolean?

(not 100% sure, but sometimes times are more useful as flags than bools)

Reply:
Agreed and changed, not needed now but could help if some logic will be needed in the future